brain-marks/Tweets/Views/TweetCard.swift

```diff
@@ -42,12 +42,30 @@ struct TweetHeaderView: View {
 struct TweetBodyView: View {
     let tweetBody: String
     var body: some View {
-        Text(tweetBody)
+        formattedText(str: tweetBody)
+            .multilineTextAlignment(.leading)
             .font(.body)
             .lineSpacing(8.0)
             .padding(EdgeInsets(top: 0, leading: 18, bottom: 18, trailing: 18))
             .fixedSize(horizontal: false, vertical: true)
     }
+    
+    private func formattedText(str: String) -> Text {
+        var result = Text("")
+        let parts = str.components(separatedBy: " ")
+        for part in parts {
+            if part.contains("#") {
```

Comment:
maaaybe by checking `if part.contains("/n")` do another split then check for `part.contains("#")`

Something like 

```
if part.contains("/n") {
     let subParts = part.components(separatedBy: "/n")
     for (subPart in subParts) {
          let newResult: Text = result + Text(" \(part)")
                     .bold()
                     .foregroundColor(.blue)
                 result = newResult
     }
} 
```

Reply:
This also looks like you want to do multiple separators which is currently not supported as far as I know, so you could try to do that to simplify the solution: https://stackoverflow.com/questions/34247316/componentsseparatedbystring-by-multiple-separators-in-swift